src/components/breadcrumb/Breadcrumb.vue

```diff
@@ -8,35 +8,53 @@
             class="back-btn"
             exact
             :href="backUrl"
-            icon small
+            icon
+            small
             :disabled="breadcrumbs.length <= 1"
           >
-            <v-icon color="primary">mdi-arrow-left</v-icon>
+            <v-icon color="primary">
+              mdi-arrow-left
+            </v-icon>
           </v-btn>
         </v-col>
 
-        <v-divider class="mx-3" color="white" vertical />
-
-        <v-col cols="auto" class="breadcrumb-col">
-          <v-breadcrumbs :items="breadcrumbs" class="pa-0 ma-0">
-            <template v-slot:divider>
-              <v-icon color="white" class="mx-n2">mdi-chevron-right</v-icon>
+        <v-divider
+          class="mx-3"
+          color="white"
+          vertical
+        />
+
+        <v-col
+          cols="auto"
+          class="breadcrumb-col"
+        >
+          <v-breadcrumbs
+            :items="breadcrumbs"
+            class="pa-0 ma-0"
+          >
+            <template #divider>
+              <v-icon
+                color="white"
+                class="mx-n2"
+              >
+                mdi-chevron-right
+              </v-icon>
             </template>
 
-            <v-breadcrumbs-item
-              slot="item"
-              slot-scope="{ item }"
-              exact
-              :to="item.to"
-              :href="item.href"
-            >
-              <span
-                class="breadcrumb-text"
-                :class="isActiveCrumb(item) ? 'active-crumb': 'inactive-crumb'"
+            <template #item="{ item }">
```

Comment:
I need to test this new syntax for the v-breadcrumb item slot.

Reply:
Works fine.

![image](https://github.com/bcgov/bcrs-shared-components/assets/10002889/d08e4805-f033-4924-8b0b-293f73a45feb)